src/main/java/com/dell/isg/smi/service/server/inventory/controller/ServerInventoryController.java

```diff
@@ -56,9 +59,7 @@
 
 
     @RequestMapping(value = "/hardware", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/hardware", nickname = "hardware", notes = "This operation allow user to get complete server hardware inventory throu wsman.", response = ServerHardwareInventory.class)
-    // @ApiImplicitParams({
-    // @ApiImplicitParam(name = "credential", value = "Credential", required = true, dataType = "Credential.class", paramType = "Body", defaultValue = "no default") })
+    @ApiOperation(value = "/hardware", nickname = "hardware", notes = "This operation allow user to get complete server hardware inventory using wsman protocol.", response = ServerHardwareInventory.class)
```

Comment:
This operation allow user to get a complete server software inventory using the wsman protocol.